layouts/shortcodes/card-code.html

```diff
@@ -1,20 +1,15 @@
-<div class="td-card card">
-  {{ $lang := "" }}
-  {{ with $.Get "lang" }}
-    {{ $lang = $.Get "lang" }}
-  {{ end }}
-  {{ $highlight := "" }}
-  {{ with $.Get "highlight" }}
-    {{ $highlight = $.Get "highlight" }}
-  {{ end }}
-  {{- with $.Get "header" -}}
+{{ $lang := default "" ($.Get "lang") -}}
```

Comment:
Consider adding at least a comment to the effect that this is deprecated.

I had missed the deprecation note int the changelog, so others might miss it too. I propose that you add a warning stating that it is deprecated and will be removed in a future release. This way projects using the shortcode will get notified when they build their site. WDYT?

Reply:
> Consider adding at least a comment to the effect that this is deprecated.

Great idea, I will do that shortly.

Reply:
Done.